packages/babel-preset-react-app/create.js

```diff
@@ -167,7 +191,7 @@ module.exports = function(api, opts, env) {
         {
           corejs: false,
           helpers: areHelpersEnabled,
-          regenerator: true,
+          regenerator: !isModern,
```

Comment:
I think it should be safe to leave this as `true` no? I imagine if we're building modern then it won't transform async/await into regenerator output?

Reply:
I think they were working on that with new preset-env and corejs.3 but I’ll revert this then, thanks for this rapid and good feedback. I appreciate the time you take to answer